src/main/java/seedu/address/logic/commands/ListTransactionCommand.java

```diff
@@ -39,7 +38,6 @@ public CommandResult execute(Model model) throws CommandException {
         }
 
         Person selected = lastShownList.get(index.getZeroBased());
-        model.updateFilteredPersonList(new IsSelectedPredicate(model, index));
```

Comment:
Remove this so the filtered person list is not changed when switching to showing transactions.

Reply:
👍